game/src/js/objects/characters/Character.js

```diff
@@ -1,48 +1,43 @@
 import Projectile from '../Projectile';
+import jsonPath from '../../util/jsonpath-0.8.0';
 
 export default class Character extends Phaser.Physics.Arcade.Sprite {
   constructor(scene, x, y, key, options = {}) {
     super(scene, x, y, key);
+    
+    //pull specific character config information from characters.json
+    //jsonPath returns query results as an array so reference 0 element to get plan config information
+    const filter=`$..characters[?(@.key =="${key}")]`;
```

Comment:
Rather than introducing a new dependency, an even leaner and more efficient way to do this is to have something like this in `characters.json`:
```javascript
{
  "knight_hero": {
    "name": "Valiant Knight",
    "key": "knight_hero",
    "scale": 0.35,
    "orientation": "E",
    "baseSpeed": 200,
    "projectile": {
      "type": "light",
      "baseRange": 1000,
      "fireOffset": {"x": 0, "y": -150}
    },
    "collider": {
      "size": 70,
      "offset": {"x": 87, "y": 80}
    }
  },
  "priest_hero": {
    "name": "Avenging Priest",
    "key": "priest_hero",
    "scale": 0.35,
    "orientation": "E",
    "baseSpeed": 200,
    "projectile": {
      "type": "orb",
      "baseRange": 1000,
      "fireOffset": {"x": 0, "y": -150}
    },
    "collider": {
      "size": 70,
      "offset": {"x": 92, "y": 80}
    }
  },
  ...
}
```
Then, getting a specific config is a matter of doing `scene.cache.json.get('characters')[key]` directly with no search.